web/src/stream_edit_subscribers.ts

```diff
@@ -263,7 +270,13 @@ function subscribe_new_users({pill_user_ids}: {pill_user_ids: number[]}): void {
         });
     }
 
-    subscriber_api.add_user_ids_to_stream(user_ids, sub, invite_success, invite_failure);
+    subscriber_api.add_user_ids_to_stream(
+        user_ids,
+        sub,
+        Boolean($("#send_notification_to_new_subscribers").prop("checked")),
```

Comment:
Using `is(":checked")` would not require `Boolean`.

Reply:
This caused a typescript error otherwise

Reply:
I don't think it will cause error when using `is(":checked")`.